docs/source/notebooks/mmm/mmm_lift_test.ipynb

```diff
@@ -0,0 +1,1716 @@
+{
```

Comment:
<p>Can you explain that yoiu are generating synthetic data with some true regression coefficients? So that the user follows the storyline of the code :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>good point</p>